README.md

```diff
@@ -27,24 +27,38 @@ are convertible to `bitcoin::PublicKey`
 completing an unsigned `bitcoin::TxIn` with appropriate data
 * Determining the specific keys, hash preimages and timelocks used to spend
 coins in a given Bitcoin transaction
-* `no_std` support enabled by disabling the `default-features` and enabling
-`"no-std"`. See `embedded/` for an example.
+* `no_std` support enabled by disabling the `default-features`. See `embedded/` for an example.
 
 More information can be found in [the documentation](https://docs.rs/miniscript)
 or in [the `examples/` directory](https://github.com/rust-bitcoin/rust-miniscript/tree/master/examples)
 
 ## Building
 
-The cargo feature `std` is enabled by default. At least one of the features `std` or `no-std` or both must be enabled.
+The cargo feature `std` is enabled by default. To enable `no_std` support you must disable default features.
 
-Enabling the `no-std` feature does not disable `std`. To disable the `std` feature you must disable default features. The `no-std` feature only enables additional features required for this crate to be usable without `std`. Both can be enabled without conflict.
+The library can be built and tested using [`cargo`](https://github.com/rust-lang/cargo/):
+
+```
+git clone git@github.com:rust-bitcoin/rust-miniscript.git
+cd rust-miniscript
+cargo build
+```
+
+You can run tests with:
+
+```
+cargo test
+```
+
+Please refer to the [`cargo` documentation](https://doc.rust-lang.org/stable/cargo/) for more
+detailed instructions.
```

Comment:
These changes are unrelated to this PR, right? (FTR I never liked these docs in `rust-bitcoin`, if one doesn't know how to run Rust unit tests they probably shouldn't be writing financial software - and `rust-miniscript` is even more hardcore than `rust-bitcoin`.

Reply:
Yeah. The **Building** section had more relevant info about how to select the `no-std`/`std` features, but now it is just one line (I added that to make it more complete).

Technically the `no_std` support is already explained in **High-Level Features** and doesn't require more in-depth explanation, so based on your comment I think it makes sense to remove **Building**.

Reply:
They're not totally unrelated because they replace docs that talked about nostd.

Though agreed that we probably don't need to tell people to run `cargo test` if there's not even any env variables or anything :). If this were end-user software then sure, but this is a Rust library that you can't use without being familiar with the language and its ecosystem.